meteor/client/ui/RundownView/RundownTiming/withTiming.tsx

```diff
@@ -6,8 +6,21 @@ import { RundownTimingContext } from '../../../../lib/rundown/rundownTiming'
 
 export type TimingFilterFunction = (durations: RundownTimingContext) => any
 
+export enum TimingTickResolution {
+	/** Once per second */
+	Synced = 0,
+	Low,
+	High,
+}
+
+export enum TimingDataResolution {
+	Low = 0,
```

Comment:
I think the naming should be Synced vs High here, to make it clear that with `TimingTickResolution.Low` and `TimingTickResolution.High` you probably should still use `TimingDataResolution.High`, and `TimingTickResolution.Synced` should go with `TimingDataResolution.Synced`

Reply:
Good idea, I've renamed that https://github.com/tv2/tv-automation-server-core/commit/0ec1c3d8876f2f5cb2915cb4fdd5b22160af181d